awscli/customizations/wizard/wizards/iam/new-role-wip.yml

```diff
@@ -233,3 +233,17 @@ execute:
         # This role_arn comes from the `output_var` in the previous
         # executor step.
         role_arn: "{role_arn}"
+__OUTPUT__:
+  value: |
+    Wizard has created a role:
+      Role name: {role_name}
+      Role ARN: {created_role_arn}
+
+    {% if output_type == preview_cli_command %}
+    Example of AWS CLI commands:
```

Comment:
Then for these lines, we can then update it to be a little more explicit here as well:
* `Steps to create role is equivalent to running the following sample AWS CLI commands:`
* `Steps to create role is equivalent to deploying the following sample AWS CloudFormation template:`